engine/Shopware/Core/sBasket.php

```diff
@@ -2624,24 +2624,58 @@ private function getAdditionalInfoForUpdateArticle(array $cartItems)
         }
 
         $sql = <<<SQL
-SELECT  s_order_basket.id, s_articles_details.minpurchase, s_articles_details.purchasesteps,
-        s_articles_details.maxpurchase, s_articles_details.purchaseunit,
-        pricegroupID, pricegroupActive, s_order_basket.ordernumber, s_order_basket.articleID,
-        GROUP_CONCAT(avoid.customergroupID SEPARATOR '|') as blocked_customer_groups
-FROM s_articles, s_order_basket, s_articles_details
-LEFT JOIN s_articles_avoid_customergroups avoid 
-  ON avoid.articleID = s_articles_details.articleID    
-WHERE s_order_basket.articleID = s_articles.id
-AND s_order_basket.ordernumber = s_articles_details.ordernumber
-AND s_order_basket.id IN (?)
-AND s_order_basket.sessionID = ?
-GROUP BY s_articles.id, s_order_basket.id
+            SELECT 
```

Comment:
Since this is a heredoc, you should probably indent this all the way to the left (i.e. the way it was before).